themes/bootstrap3/js/facets.js

```diff
@@ -251,11 +220,11 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
 
       let icon = elem.querySelector('.icon');
       if (icon !== null) {
-        if (!areCheckboxClassesDefined()) {
-          loadCheckboxClasses();
-        }
-        icon.classList.toggle(checkboxClasses.checked);
-        icon.classList.toggle(checkboxClasses.unchecked);
+        let checked = icon.attributes['data-checked'].value === 'true';
+        let attrs = [];
```

Comment:
Should you initialize to `{}` rather than `[]` since this is an object rather than an array?